### PR TITLE
Cast trampolines before transmuting

### PR DESCRIPTION
**Pull request by EPashkin**
Fix warning introduced https://github.com/rust-lang/rust/issues/19925
See https://github.com/gtk-rs/gtk/issues/278

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
Good too. Thanks @EPashkin!

File: src/signal.rs [hunk 1]
```diff
@@ -90,7 +90,7 @@ pub fn idle_add<F>(func: F) -> u32
     where F: FnMut() -> Continue + 'static {
     assert_initialized_main_thread!();
     unsafe {
-        glib_ffi::g_idle_add_full(glib_ffi::G_PRIORITY_DEFAULT_IDLE, transmute(trampoline),
+        glib_ffi::g_idle_add_full(glib_ffi::G_PRIORITY_DEFAULT_IDLE, transmute(trampoline as usize),
```
Comment (gkoz) [inline src/signal.rs]: We can apply to these the same approach as in `glib` right?

**Comment by EPashkin** [comment]
Updated 3 first functions, but others still needed `transmute` as `connect` accept only `GCallback` (function without parameters)

**Comment by gkoz** [comment]
Thanks!

**Pull request merged**